### PR TITLE
fix(preflight): the temp-path lane catches bare-root mkdtemp forms

### PR DESCRIPTION
**Pull request by bmethod**
Follow-up from the drovr#570 re-vendor review (the one finding that survived triage): `tempfile.mkdtemp(dir="/tmp")` — and any mkdtemp-family call carrying a bare quoted temp root — creates a fresh subdirectory under the literal root (or a root-level `/tmpXXXXXX` sibling for the prefix form), which is exactly the leak shape the lane exists for, and the trailing-slash requirement missed it.

The mkdtemp family always creates, so a new alternation matches `mkdtemp`/`mkdtempSync` with a bare quoted `/tmp` or `/var/tmp` anywhere in the arguments; plain `mkdir` of the bare root stays excluded (creating `/tmp` itself is a no-op — the pinned decision from #1483's review). The line fast-path widens from `/tmp/` to `/tmp` accordingly. Also rewords the fixture comment the same review flagged as ambiguous.

Tests: new must-fire pin for the keyword form; all four preflight suites pass; size-ratchet row bumped in-diff.

https://claude.ai/code/session_01SgCfsq4oKxc45iq25VcPhG

### CODE REVIEW

File: skills/preflight/scripts/preflight [hunk 1]
```diff
@@ -926,7 +931,7 @@ while IFS= read -r f; do
     # nothing.
     if [ "$is_md" = 0 ]; then
       case "$content" in
-        *"/tmp/"*)
+        *"/tmp"*)
```
[the remark this answers is not in the document]
Comment (bmethod) [reply skills/preflight/scripts/preflight:934]: Declined — the case guard is a cost gate, not a matcher: its only job is to keep the pipeline from spawning on the overwhelming majority of lines, and precision lives entirely in the regex behind it, which requires an opening quote before the root and a closing delimiter (`/` or quote) after — a `/tmpfoo` line spawns one pipeline that the regex rejects, emitting nothing. Tightening the guard to re-encode the regex's boundary conditions duplicates the matcher into the fast path, which is exactly the drift hazard a dumb guard avoids, to save a no-op pipeline on lines that round to zero.

File: skills/preflight/tests/lanes-must-fail.test.sh [hunk 2]
```diff
@@ -121,11 +121,12 @@ fires "an mktemp with no arguments is the same finding" "scripts/scratchfile.sh:
...
 mkdir -p "$R/src"
-# Trip paths are substituted at run time, so this suite's committed bytes
-# never join a creation call to a temp-path literal.
+# Trip paths are substituted at run time: the generated FIXTURE repo carries
+# the literals by design, while this suite's own committed bytes never join
+# a creation call to one.
 printf '#!/usr/bin/env bash\nset -euo pipefail\nmkdir -p %s/cache\n' /tmp >"$R/scripts/shellmk.sh"
 printf 'const fs = require("fs");\nfs.mkdirSync("%s/out");\nfs.mkdtempSync("%s/app-");\n' /tmp /tmp >"$R/src/mk.js"
-printf 'import os, tempfile\nos.makedirs("%s/state")\ntempfile.mkdtemp(dir="%s/keep")\n' /tmp /tmp >"$R/src/mk.py"
+printf 'import os, tempfile\nos.makedirs("%s/state")\ntempfile.mkdtemp(dir="%s/keep")\ntempfile.mkdtemp(dir="%s")\n' /tmp /tmp /tmp >"$R/src/mk.py"
```
[the remark this answers is not in the document]
Comment (bmethod) [reply skills/preflight/tests/lanes-must-fail.test.sh:129]: Applied — `fs.mkdtempSync("/tmp")` joins the must-fire fixtures with its own assertion.

File: skills/preflight/scripts/preflight [hunk 3]
```diff
@@ -547,6 +547,11 @@ has_exit_trap() { # FILE
 # real creation sites, and a lane that flags the fixtures is noise that
 # buries the finding.
 TMPPATH_CREATE_RE='(mkdtemp|mkdtempSync|mkdir|mkdirSync|makedirs|create_dir_all)[[:space:]]*\([^,)]*["'\''`]/(var/)?tmp/'
+# The mkdtemp family ALWAYS creates, so a bare quoted temp root anywhere in
+# its arguments is the same leak (tempfile.mkdtemp(dir="/tmp")) — or worse,
+# a root-level sibling (mkdtempSync("/tmp") makes /tmpXXXXXX). Plain mkdir
+# of the bare root stays out: creating /tmp itself is a no-op.
+TMPPATH_CREATE_RE="$TMPPATH_CREATE_RE"'|(mkdtemp|mkdtempSync)[[:space:]]*\([^)]*["'\''`]/(var/)?tmp["'\''`]'
 TMPPATH_CREATE_RE="$TMPPATH_CREATE_RE"'|(^[[:space:]]*|[;|&(`{][[:space:]]*)mkdir[[:space:]]+-p[[:space:]]+["'\'']?/(var/)?tmp/'
```
[the remark this answers is not in the document]
Comment (bmethod) [reply skills/preflight/scripts/preflight:555]: Declined — the concatenation idiom predates this diff (the pattern's first two alternations already build this way); the new alternation extends it rather than introducing it. Restyling the construction is its own change with its own risk, and the pattern's correctness is pinned where it matters: `bash -n` plus seven must-fire/must-not-fire fixtures that break on any quoting regression.
